app/models/event.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class Event < ApplicationRecord
+  scope :upcoming, ->(n = Date.today.to_s) { where('date > ?', n) }
+  scope :past, ->(n = Date.today.to_s) { where('date < ?', n) }
+  has_many :invitations
+  has_many :users, through: :invitations
+
```

Comment:
- [ ] the event model is created by one user only, where is that represented? The event model should have one [creator](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_assoc_requirement.png).